### PR TITLE
Nfdiv 1668

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-1668

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: config/default.yaml [hunk 1]
```diff
@@ -37,8 +37,8 @@ services:
     systemUsername: 'dummy_user'
     systemPassword: 'dummy_password'
   case:
-    url: 'http://ccd-data-store-api-aat.service.core-compute-aat.internal'
-    #url: 'http://ccd-data-store-api-nfdiv-case-api-pr-1348.service.core-compute-preview.internal'
+    #    url: 'http://ccd-data-store-api-aat.service.core-compute-aat.internal'
+    url: 'http://ccd-data-store-api-nfdiv-case-api-pr-1550.service.core-compute-preview.internal'
```
Comment (linusnorton) [inline config/default.yaml:41]: assuming you'll revert this

**Review by linusnorton — APPROVED** [review]

File: src/main/steps/home/get.test.ts [hunk 2]
```diff
@@ -681,6 +683,44 @@ describe('HomeGetController', () => {
     expect(res.redirect).toBeCalledWith(`${RESPONDENT}${HUB_PAGE}`);
   });
 
+  test('redirects to the review the application for respondent users in holding state and aos is not completed', () => {
```
Comment (lara-mpt) [inline src/main/steps/home/get.test.ts:686]: could call this '... and aos is not started'

File: src/main/steps/home/get.test.ts [hunk 3]
```diff
@@ -681,6 +683,44 @@ describe('HomeGetController', () => {
...
+        isApplicant2: true,
+      },
+    });
+    const res = mockResponse();
+    controller.get(req, res);
+
+    expect(res.redirect).toBeCalledWith(`${RESPONDENT}${REVIEW_THE_APPLICATION}`);
+  });
+
+  test('redirects to the CYA for respondent users in holding state and aos is not completed but the first question is complete', () => {
```
Comment (lara-mpt) [inline src/main/steps/home/get.test.ts:704]: this would be '... and aos is started'

File: src/main/steps/home/get.ts [hunk 4]
```diff
@@ -43,7 +46,18 @@ export class HomeGetController {
     const isFirstQuestionComplete = firstQuestionForm.getErrors(req.session.userCase).length === 0;
 
     if (req.session.isApplicant2 && req.session.userCase.applicationType === ApplicationType.SOLE_APPLICATION) {
-      res.redirect(respondentRedirectPageSwitch(req.session.userCase.state, isFirstQuestionComplete));
+      const checkYourAnswersForm = new Form(<FormFields>respondentCYAForm.fields);
+      const isLastQuestionComplete = checkYourAnswersForm.getErrors(req.session.userCase).length === 0;
+      const reviewedApplicationForm = new Form(<FormFields>reviewApplicationQuestionForm.fields);
+      const hasReviewTheApplication = reviewedApplicationForm.getErrors(req.session.userCase).length === 0;
```
Comment (lara-mpt) [inline src/main/steps/home/get.ts:52]: hasReviewTheApplication should be hasReviewedTheApplication ?

**Pull request closed without merging**